test/DynamicEater.test.ts

```
@@ -0,0 +1,86 @@
+
+		const mergeAddress = await merge.getAddress();
+
+		// Arrange
+		await wewe.approve(mergeAddress, ethers.parseUnits("1000000", 18));
+		await token.connect(otherAccount).approve(mergeAddress, ethers.parseUnits("1000000", decimals));
+
+		return { owner, otherAccount, token, wewe, merge };
+	}
+
+	describe("Merge with dynamic rates", () => {
+		it.only("Should get price as percent at start of bonding curve", async () => {
```

Biome

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)